src/redux/categories/categories.js

```diff
@@ -0,0 +1,19 @@
+import { createSlice } from '@reduxjs/toolkit';
+
+const initialState = {
+  categories: '',
+};
+
+const categories = createSlice({
+  name: 'categories',
+  initialState,
+  reducers: {
+    checkStatus: (state, action) => ({
+      ...state,
+      categories: action.payload,
+    }),
+  },
+});
+
+export const { checkStatus } = categories.actions;
```

Comment:
- [ ] Kindly be noticed that there should be a `CHECK_STATUS` as an action type, The value of the action type needs to follow the [ducks pattern](https://github.com/erikras/ducks-modular-redux), so it will be something like: `const CHECK_STATUS = bookstore/categories/CHECK_STATUS`.
- [ ] Also you should create and export **Action Creator**. (`checkStatus` function).

Reply:
Many thanks for the feedback @ShahierNashaat! All the changes have been applied